src/lib/y2network/config.rb

```diff
@@ -34,30 +34,28 @@ module Y2Network
   #   config.routing.tables.first << route
   #   config.write
   class Config
-    # @return [Symbol] Configuration ID
-    attr_reader :id
     # @return [Array<Interface>]
-    attr_reader :interfaces
+    attr_accessor :interfaces
     # @return [Routing]
     attr_reader :routing
     # @return [Symbol] Information source (see {Y2Network::Reader} and {Y2Network::Writer})
     attr_reader :source
 
     class << self
       # @param source [Symbol] Source to read the configuration from
-      def from(source)
-        reader = ConfigReader.for(source)
+      # @param opts   [Hash]   Reader options
```

Comment:
I think this will need a bit more description, like what options is supported and so on. Now I have now idea what it is good for.